doc/README.md

```diff
@@ -251,25 +251,42 @@ You, as the writer of documentation, are still in charge of its content.
   For example:
 
   ```markdown
-  # pkgs.coolFunction
+  # pkgs.coolFunction {#pkgs.coolFunction}
 
-  Description of what `coolFunction` does.
+  `pkgs.coolFunction` *`name`* *`config`*
 
-  ## Inputs
+  Description of what `callPackage` does.
 
-  `coolFunction` expects a single argument which should be an attribute set, with the following possible attributes:
 
-  `name` (String)
+  ## Inputs {#pkgs-coolFunction-inputs}
+
+  If something's special about `coolFunction`'s general argument handling, you can say so here.
+  Otherwise, just describe the single argument or start the arguments' definition list without introduction.
+
+  *`name`* (String)
```

Comment:
```suggestion
  `name` (String)
```

I've mostly not seen the *bold* style

Reply:
https://nixos.org/manual/nixpkgs/stable/#function-library-lib.fileset.traceVal

Reply:
It's consistent with the Nix manual, e.g. [`builtins` page](https://nix.dev/manual/nix/2.23/language/builtins), [operators](https://nix.dev/manual/nix/2.23/language/operators), and [derivation attributes](https://nix.dev/manual/nix/2.23/language/advanced-attributes).